tests/Pheanstalk/BugfixConnectionTest.php

```diff
@@ -27,7 +27,7 @@ public function testIssue4ReadingOver8192Bytes()
         $pheanstalk = $this->createPheanstalk();
         $pheanstalk->put(str_repeat('.', $length));
         $job = $pheanstalk->peekReady();
-        $this->assertEquals(strlen($job->getData()), $length, 'data length: %s');
+        self::assertEquals(strlen($job->getData()), $length, 'data length: %s');
```

Comment:
```suggestion
        self::assertSame(strlen($job->getData()), $length, 'data length: %s');
```
(always prefer strict type assertions over loose type)

Reply:
Note: not sure why the change was made from the `$this->` pattern to the `self::` pattern, but just FYI: both will work with the PHPUnit Polyfills.

Reply:
It was because they are apparently static in newer versions of phpunit, and my IDE was complaining about them.

Reply:
Ah! Well, those methods were always static, might just be that your IDE got smarter 😂